frontend/src/app/utils/disease.ts

```diff
@@ -8,5 +8,27 @@ export const useSupportedDiseaseList = () => {
   if (!hivEnabled) {
     allDiseases = allDiseases.filter((d) => d !== MULTIPLEX_DISEASES.HIV);
   }
+  const syphilisEnabled = Boolean(useFeature("syphilisEnabled"));
+  if (!syphilisEnabled) {
+    allDiseases = allDiseases.filter((d) => d !== MULTIPLEX_DISEASES.SYPHILIS);
+  }
   return allDiseases;
 };
+
+export const useDisabledFeatureDiseaseList = () => {
+  const allDiseases = Object.values(MULTIPLEX_DISEASES);
+  const supportedDiseases = useSupportedDiseaseList();
+  return allDiseases.filter((d) => !supportedDiseases.includes(d));
+};
+
+export const useSupportedDiseaseOptionList = () => {
```

Comment:
could we move this method into the file that it's being imported? This function seems tailor made for the specific result selector page and think it might be a bit more readable to have the code that formats the data for the UI next to where it's consumed.

Reply:
We could although I made this into a separate hook since the dashboard in `Analytics.tsx` also uses an identical option list for supported diseases so in the future we could update it to use this hook as well. Thoughts?

Reply:
yeah in that case if we could move the hook itself into a UI-forward page (either in `TestFiltersList.tsx` or `Analytics.tsx`) and then export it from there, that'd be good. Was just somewhat confused just reading the util file by itself what the point of this function was, but if it was right next to the piece of UI it's being consumed in, don't think we'd have that problem.

Reply:
Hmm I feel like it makes more sense to store it in a util file since it's a custom hook rather than inside an individual component and importing it from that component into another. For these types of utils I don't think we'll often be reading the util file itself first and then determining the context in which it's being used, I think it's more often the reverse where it's used as a custom hook in a component and then if you want to see how it's implemented you can click/follow through to see